tests/test_persisted_task_properties.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for validating persisted task properties for different data types."""
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+
+
+class TestPersistedTaskPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for
+    persisted task properties for different data types.
+    """
```

Comment:
Remove the test class and use test functions.

Reply:
removed